etl/steps/data/garden/gcp/2022-09-29/global_carbon_budget_additional.countries.json

```diff
@@ -0,0 +1,235 @@
+{
+  "Afghanistan": "Afghanistan",
+  "Africa": "Africa (GCP)",
+  "Albania": "Albania",
+  "Algeria": "Algeria",
+  "Andorra": "Andorra",
+  "Angola": "Angola",
+  "Anguilla": "Anguilla",
+  "Antigua and Barbuda": "Antigua and Barbuda",
+  "Argentina": "Argentina",
+  "Armenia": "Armenia",
+  "Aruba": "Aruba",
+  "Asia": "Asia (GCP)",
+  "Australia": "Australia",
+  "Austria": "Austria",
+  "Azerbaijan": "Azerbaijan",
+  "Bahamas": "Bahamas",
+  "Bahrain": "Bahrain",
+  "Bangladesh": "Bangladesh",
+  "Barbados": "Barbados",
+  "Belarus": "Belarus",
+  "Belgium": "Belgium",
+  "Belize": "Belize",
+  "Benin": "Benin",
+  "Bermuda": "Bermuda",
+  "Bhutan": "Bhutan",
+  "Bolivia": "Bolivia",
+  "Bonaire, Saint Eustatius and Saba": "Bonaire Sint Eustatius and Saba",
+  "Bosnia and Herzegovina": "Bosnia and Herzegovina",
+  "Botswana": "Botswana",
+  "Brazil": "Brazil",
+  "British Virgin Islands": "British Virgin Islands",
+  "Brunei Darussalam": "Brunei",
+  "Bulgaria": "Bulgaria",
+  "Burkina Faso": "Burkina Faso",
+  "Burundi": "Burundi",
+  "Cambodia": "Cambodia",
+  "Cameroon": "Cameroon",
+  "Canada": "Canada",
+  "Cape Verde": "Cape Verde",
+  "Central African Republic": "Central African Republic",
+  "Chad": "Chad",
+  "Chile": "Chile",
+  "China": "China",
+  "Colombia": "Colombia",
+  "Comoros": "Comoros",
+  "Congo": "Congo",
+  "Cook Islands": "Cook Islands",
+  "Costa Rica": "Costa Rica",
+  "Croatia": "Croatia",
+  "Cuba": "Cuba",
+  "Cura\u00e7ao": "Curacao",
+  "Cyprus": "Cyprus",
+  "Czech Republic": "Czechia",
+  "C\u00f4te d'Ivoire": "Cote d'Ivoire",
+  "Democratic Republic of the Congo": "Democratic Republic of Congo",
+  "Denmark": "Denmark",
+  "Djibouti": "Djibouti",
+  "Dominica": "Dominica",
+  "Dominican Republic": "Dominican Republic",
+  "EU27": "European Union (27) (GCP)",
```

Comment:
Is the GCP definition of the EU27 different from ours? Could it not be just 'European Union (27)'?

Reply:
They actually do differ, I'm looking into it, there is something strange there...